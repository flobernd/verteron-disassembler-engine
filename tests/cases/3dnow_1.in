-64 0F 0F 01 B6 0F 0F 46 6C A6 0F 0F A7 DB AB A2