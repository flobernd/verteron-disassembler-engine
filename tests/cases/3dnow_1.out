== [    BASIC ] ============================================================================================
   MNEMONIC: pfrcpit2 [ENC: 3DNOW, MAP: 0F0F, OPC: 0xB6]
     LENGTH:  4
        SSZ: 64
       EOSZ: 32
       EASZ: 64
   CATEGORY: AMD3DNOW
    ISA-SET: AMD3DNOW
    ISA-EXT: AMD3DNOW
 EXCEPTIONS: NONE
 ATTRIBUTES: HAS_MODRM 

== [ OPERANDS ] ============================================================================================
##       TYPE  VISIBILITY  ACTION      ENCODING   SIZE  NELEM  ELEMSZ  ELEMTYPE                        VALUE
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------
 0   REGISTER    EXPLICIT      RW     MODRM_REG     64      1      64       INT                          mm0
 1     MEMORY    EXPLICIT       R      MODRM_RM     64      1      64       INT  TYPE  =                 MEM
                                                                                 SEG   =                  ds
                                                                                 BASE  =                 rcx
                                                                                 INDEX =                none
                                                                                 SCALE =                   0
                                                                                 DISP  =  0x0000000000000000
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------

== [      ATT ] ============================================================================================
   ABSOLUTE: pfrcpit2 (%rcx), %mm0
   RELATIVE: pfrcpit2 (%rcx), %mm0

== [    INTEL ] ============================================================================================
   ABSOLUTE: pfrcpit2 mm0, qword ptr ds:[rcx]
   RELATIVE: pfrcpit2 mm0, qword ptr ds:[rcx]

== [ SEGMENTS ] ============================================================================================
0F 0F 01 B6 
:     :  :..OPCODE
:     :..MODRM
:..OPCODE
